systemd/units/init.sls

```diff
@@ -1,10 +1,10 @@
-{% for unittype, units in pillar['systemd:units'].iteritems()  %}
+{% for unittype, units in pillar['systemd'].iteritems()  %}
```

Comment:
I miss example in example Pilař or readme. 

Quite sure this First line si wrong. I Ha logical sena to have

Systemd:
     Unit:{}
     Network:{}

Reply:
pillar.example is there (without units) and it works as expected. Not sure what is missing.

Reply:
I tested it on debian-9 with vagrant and also in production. No problem on both